Code/Tools/ProjectManager/Platform/Windows/ProjectUtils_windows.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ProjectUtils.h>
+
+#include <QDir>
+#include <QFileInfo>
+#include <QProcess>
+#include <QProcessEnvironment>
+
+namespace O3DE::ProjectManager
+{
+    namespace ProjectUtils
+    {
+        AZ::Outcome<void, QString> FindSupportedCompilerForPlatform()
+        {
+            QProcessEnvironment environment = QProcessEnvironment::systemEnvironment();
+            QString programFilesPath = environment.value("ProgramFiles(x86)");
+            QString vsWherePath = QDir(programFilesPath).filePath("Microsoft Visual Studio/Installer/vswhere.exe");
+
+            QFileInfo vsWhereFile(vsWherePath);
+            if (vsWhereFile.exists() && vsWhereFile.isFile())
+            {
+                QProcess vsWhereProcess;
+                vsWhereProcess.setProcessChannelMode(QProcess::MergedChannels);
+
+                vsWhereProcess.start(
+                    vsWherePath,
+                    QStringList{
+                        "-version",
+                        "16.0",
+                        "-latest",
+                        "-requires",
+                        "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+                        "-property",
+                        "isComplete"
+                    });
+
+                if (vsWhereProcess.waitForStarted())
+                {
+                    while (vsWhereProcess.waitForReadyRead()) {}
```

Comment:
A sleep() or usleep() might be appropriate in this while loop

Reply:
Replaced with waitForFinished